node/runtime/src/lib.rs

```diff
@@ -60,8 +60,8 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("node"),
 	impl_name: create_runtime_str!("substrate-node"),
 	authoring_version: 10,
-	spec_version: 33,
-	impl_version: 33,
+	spec_version: 34,
```

Comment:
These changes are not required, you did not change the implementation nor the interface. I know that the CI will complain, but we can ignore this.

Reply:
Revert it

Reply:
ty